src/components/home/header.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import logo from '../../images/logo_white.png';
```

Comment:
@noor-ahmed Can we set alias from `../../images/` to `Images/` in the configuration? 🤔